helm/charts/td-agent-barito/values.yaml

```diff
@@ -15,8 +15,10 @@ rbac:
 resources:
   limits:
     memory: 1Gi
+    ephemeral-storage: 6Gi
```

Comment:
@BimoZX why is requesting quite big 6Gi?

Reply:
I'm just playing it safe by having double the request value on the limit. I think it's fine if it's a little bit too high, cause we don't want to risk the pod being evicted.

Reply:
okay then